packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -122,46 +77,25 @@ export class EthereumAnchorService implements AnchorService {
     // Do Nothing
   }
 
-  async init(): Promise<void> {
+  async init(store: AnchorRequestStore, eventHandler: AnchorLoopHandler): Promise<void> {
+    this.#store = store
+    // Get the chainIds supported by our anchor service
     const supportedChains = await this.#cas.supportedChains()
+    if (supportedChains.length > 1) {
```

Comment:
Thinking about this more, we could just do

```
if (supportedChains.length != 1) {
  throw new NotSingleChainError()
}```

Reply:
Okay. Changed.